src/dashboard/src/components/administration/forms.py

```diff
@@ -129,10 +139,306 @@ class AtomDipUploadSettingsForm(SettingsForm):
         help_text="Show additional details.",
         choices=((False, 'No'), (True, 'Yes')))
 
-class TaxonomyTermForm(ModelForm):
+
+class TaxonomyTermForm(forms.ModelForm):
     class Meta:
         model = models.TaxonomyTerm
         fields = ('taxonomy', 'term')
         widgets = {
             "term": TextInput(attrs=settings.INPUT_ATTRS)
         }
+
+
+class ProcessingConfigurationForm(forms.Form):
+    """
+    Build processing configuration form bounded to a processingMCP document.
+    """
+
+    # The available processing fields indexed by choice_uuid.
+    processing_fields = {
```

Comment:
Idea: It would be nice if the processing config options were in the same order they came up in processing.  If this was an [OrderedDict](https://docs.python.org/2/library/collections.html#collections.OrderedDict) and the options were reordered, would that happen?